server.js

```diff
@@ -985,6 +984,48 @@ routes['join_room'] = function (user, data) {
     room.sync(user);
 }
 
+routes['search_youtube'] = function (user, data) {
+	if(user.room === null) return;
+	var room = user.room;
+
+	var query = encodeURIComponent(data.query);
+
+
+	
+	if (typeof(data.query !== query) || query.length < 1 || query.length > 100) {
```

Comment:
No, as that would be confusing behavior. I meant trim the input before testing the length, as the user may have entered a space and pressed enter. This could cause the search to fail with a confusing error message.
You've already fixed this though.